main.go

```diff
@@ -4,7 +4,6 @@ import (
 	"github.com/datarootsio/cheek/cmd"
 )
 
-//go:generate npm run build
```

Comment:
Why don't we do this build step anymore? This was done so we have a fresh build of tailwind at every binary build.

Reply:
That was removed by mistake. I was deleting a few lines of code that I had written, I will put it back. No changes were made to the main.go.